travis/push-to-cicd-status.sh

```diff
@@ -78,7 +81,10 @@ git_add_commit_push() {
     fi
     git add .
     if [[ ${TRAVIS_REPO_SLUG##*/} != "acc-provision" ]]; then
-        git commit -a -m "${RELEASE_Z_TAG}-${IMAGE}-${TRAVIS_BUILD_NUMBER}-$(date '+%F_%H:%M:%S')" -m "Commit: ${TRAVIS_COMMIT}" -m "Tags: ${IMAGE_Z_TAG}, ${TRAVIS_TAG_WITH_UPSTREAM_ID_DATE_TRAVIS_BUILD_NUMBER}" -m "${IMAGE_SHA}"
+        docker image inspect quay.io/noiro/${IMAGE}:${IMAGE_Z_TAG}
```

Comment:
Is this required?

Reply:
i had it just for log purpose in travis build to see the actual image artifcts that we push. We can remove this